public/blog/data.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "title": "How (and When) to use Star History",
+    "slug": "how-to-use-star-history",
+    "author": "Mila",
+    "featured": true,
+    "featureImage": "/blog/assets/how-to-use-star-history/how-to-use-star-history.webp",
```

Comment:
need to change from "how-to-use-star-history" to "how-to-use-github-star-hstory"